guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -33,24 +35,58 @@ the upgrade to proceed.
 +
 For information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
 
-. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
-
-. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
+ifdef::katello[]
+. Update repositories for el7
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-installer} --foreman-proxy-dns-managed=false \
---foreman-proxy-dhcp-managed=false
+# yum update -y http://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+----
+. Update repositories for el8
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y http://yum.theforeman.org/katello/{KatelloVersion}/katello/el8/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el8/x86_64/foreman-release.rpm
 ----
-
-. In the {Project} web UI, go to *Hosts* > *Discovered hosts*. If there are discovered hosts available, power off the hosts and then delete all entries under the `Discovered hosts` page. Select all other organizations in turn using the organization setting menu and repeat this action as required. Reboot these hosts after the upgrade has completed.
-
 . Clean yum cache:
 +
 ----
 # yum clean all
 ----
-
++
+. Update Packages:
++
+----
+# yum update -y
+----
++
+. Regenerate certificates.  On the main Foreman/Katello server:
++
+----
+# foreman-proxy-certs-generate --foreman-proxy-fqdn "myproxy.example.com"\
```

Comment:
This may be me, but I prefer a space before the backslash to clearly separate it.
```suggestion
# foreman-proxy-certs-generate --foreman-proxy-fqdn "myproxy.example.com" \
```

I'd also keep the downtime window smaller and do this first, then upgrade the actual content proxy. Then you have everything in place.